src/controllers/party.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable prefer-const */
+import Validate from '../middlewares/helper';
+import Party from '../models/party';
+
+const PartyController = {
+  createParty(request, response) {
+    if (Validate.spaces(request.body)) {
+      return response.status(400).send({
+        status: 400,
+        error: 'Fields should contain actual characters and not only spaces',
+      });
+    }
+    const address = request.hqAddressUrl;
+    let {
+      id, name, hqAddressUrl, logoUrl, status, createdOn, createdBy,
```

Comment:
Following the format on the ADC docs a party should have an `hqAddress` representative of the address of the party's headquarters not `hqAddressUrl`. Take note pls
<img width="544" alt="screen shot 2019-01-29 at 12 59 59 am" src="https://user-images.githubusercontent.com/26818808/51874497-7c8e2100-2361-11e9-8d78-0d158537f10e.png">